rogue-thi-app/components/modal/FilterFoodModal.jsx

```diff
@@ -84,6 +86,34 @@ export default function FilterFoodModal () {
             />
           </div>
 
+          <hr/>
+          <div>
+            <h6>
+              {t('food.filterModal.languageFood.title')}
+            </h6>
+          </div>
+          <Form.Check
+            id='LanguageFoodDefault'
+            label={t('food.filterModal.languageFood.default')}
+            checked={selectedLanguageFood === 'default' || selectedLanguageFood.length === 0}
+            onChange={() => toggleSelectedLanguageFood('default')}
+            type={'radio'} // ohne type ist es eine checkbox
+          />
+          <Form.Check
```

Comment:
In the best case, these options should be created dynamically based on the supported languages (see [`LanguageModal.jsx`](https://github.com/neuland-ingolstadt/neuland.app/blob/df2131b72550aa3b445262fc4449983f7920740e/rogue-thi-app/components/modal/LanguageModal.jsx#L18)).